config/locales/en.yml

```diff
@@ -407,6 +407,22 @@ en:
       interest_topics: "Click to start <a href='%{url}' class='form-grey' style='text-decoration: underline;'>following</a> topics that interest you"
       welcome: "Welcome to the new dashboard!"
       all_new_activity: "<a href='%{url}' class='form-grey'>All new activity</a>"
+       locations:
+    form:
+     scale: "Scale"
+      world: "World"
+      building: "Building"
+      search_name: "Search by place name"
+      coordinates: 'Or <a href="#" id="coord_button">by entering coordinates</a>'
+      latitude: "Latitude"
+      longitude: "Longitude"
+     current_location: "Use current location"
+      blur_location: "Blur my location"
+     learn_more: "Learn more »"
+    location_saved: "How this location will be saved"
+      save: "Save"
+    modal:
+    location: "Enter a location"
```

Comment:
Hi @anayo-hub, the key-value pairs are not indented properly. Take a look at the issue to see how they should be indented. Thanks!

Reply:
should i restart it afresh, to affect the changes @TildaDares

Reply:
hi @anayo-hub no you just need to modify the file and push the comment to "patch-3" branch on your forked repo, and tell us if you need help 😄

Reply:
@anayo-hub To modify the file as @DalilaBA said, you navigate to this link https://github.com/anayo-hub/plots2/edit/patch-3/config/locales/en.yml?pr=%2Fpubliclab%2Fplots2%2Fpull%2F10462 and indent those lines properly.

Reply:
@DalilaBA and @TildaDares i have done it, please review

Reply:
Hi @anayo-hub, it's not quite there yet. You just have to follow how the `like` key on line 426 was indented. Thanks!